pkg/processor/helmProcessor.go

```diff
@@ -125,14 +126,42 @@ func (h HelmProcessor) Generate(input *string, basePath string, path string) (*s
 	if err != nil {
 		return nil, err
 	}
-	helmValues, err := features.GetHelmValues()
+	rawHelmValues, err := features.GetHelmValues()
 	if err != nil {
 		return nil, err
 	}
-	err = MergeYaml(path+"/"+helmValues[0], features.GetHelmMerge(), features.GetHelmPatch())
-	if err != nil {
-		return nil, err
+
+	helmValues := []string{}
+	for _, rawHelmValueFile := range rawHelmValues {
+		if features.GetHelmIgnoreMissingValueFiles() {
+			_, err = os.Stat(path + "/" + rawHelmValueFile)
+			if !os.IsNotExist(err) {
+				helmValues = append(helmValues, rawHelmValueFile)
+			}
+		} else {
+			helmValues = append(helmValues, rawHelmValueFile)
+		}
+	}
+
+	if features.GetHelmMerge() != `` || features.GetHelmPatch() != `` {
+		var mergeTarget string
+		if len(helmValues) > 0 {
+			mergeTarget = helmValues[0]
+		} else {
+			// Special case: if you specify non-default values files, but none
```

Comment:
I don't want to revive that behaviour - it's misleading in actual use, so I'd like to remove this section and not magic up a values.yaml at this point.

Reply:
Sorry for the delay on replying. I'd be up for a `LOVELY_HELM_VALUES_YAML` as a separate feature - it does make more sense than `_MERGE` - in practice I found `_MERGE` could be misleading if you'd somehow managed to not check in values.yaml or misspelt its name.